templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,20 @@
 # Press Enter to send
```

Comment:
Let's make this have the same title as `index.md` by titling this `# Enable or disable pressing enter to send`